e2e/cloudfront/cloudfront_test.go

```diff
@@ -98,6 +121,52 @@ var _ = Describe("CloudFront", func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 
+		It("updating s3 bucket policy should succeed", func() {
+			out, err := cli.EnvShow(&client.EnvShowRequest{
+				AppName: appName,
+				EnvName: "test",
+			})
+			Expect(err).NotTo(HaveOccurred())
+			var accountID string
+			var cfDistID string
+			for _, resource := range out.Resources {
+				switch resource["logicalID"] {
+				case "PublicLoadBalancer":
+					parsed, err := arn.Parse(resource["physicalID"])
+					Expect(err).NotTo(HaveOccurred())
+					accountID = parsed.AccountID
```

Comment:
lol that's a funny way of getting the account ID, is it not available in the physical ID of the CF Distribution?

Reply:
lol i wish the physical ID could be the full arn but no it is just the CF distribution ID.